lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -20,16 +21,25 @@ class TestBlocksView(SharedModuleStoreTestCase):
     """
     Test class for BlocksView
     """
-    requested_fields = ['graded', 'format', 'student_view_multi_device', 'children', 'not_a_field']
+    requested_fields = ['graded', 'format', 'student_view_multi_device', 'children', 'not_a_field', 'due']
     BLOCK_TYPES_WITH_STUDENT_VIEW_DATA = ['video', 'discussion']
 
     @classmethod
     def setUpClass(cls):
         super(TestBlocksView, cls).setUpClass()
 
         # create a toy course
-        cls.course_key = ToyCourseFactory.create().id
+        cls.course = ToyCourseFactory.create(
+            modulestore=cls.store,
+            due=datetime(3013, 9, 18, 11, 30, 00),
+            org=ToyCourseFactory.org,
+            course=ToyCourseFactory.course,
+            run=ToyCourseFactory.run,
+            display_name=ToyCourseFactory.display_name,
+        )
```

Comment:
Nit: Can you remove the `org`, `course`, `run`, and `display_name` parameters in this call since they are just using the factory's defaults?